Functions.Templates/Templates-v2/DaprPublishOutputBinding-Python/blueprint_body.py

```diff
@@ -0,0 +1,26 @@
+
+@$(BLUEPRINT_FILENAME).schedule(schedule="*/10 * * * * *", arg_name="myTimer", run_on_startup=True)
```

Comment:
Could you rename schedule to timer_trigger in all places
@$(BLUEPRINT_FILENAME).timer_trigger(schedule="*/10 * * * * *", arg_name="myTimer", run_on_startup=True)